src/components/opoTable/opoTable.js

```diff
@@ -10,18 +11,40 @@ import Tier from "../tier/tier";
 
 import * as styles from "./opoTable.module.css";
 
-export default function OpoTable({ headings, inState = true, opos, title }) {
+export default function OpoTable({
+  headings,
+  inState = true,
+  inOpo = false,
+  opos,
+  title,
+}) {
   const columns = useMemo(() => {
     const cols = inState
       ? ["name", "region", "tier", "donors", "shadow", "investigation"]
+      : inOpo
+      ? ["ethnicity", "death", "donors", "recovery", "rank"]
       : ["states", "name", "tier", "donors", "shadow"];
 
     const createCol = accessor => {
       const col = {
         Header: <ReactMarkdown>{headings[accessor]}</ReactMarkdown>,
         accessor,
       };
-      if (accessor === "donors" || accessor === "investigation") {
+      if (accessor === "name") {
+        return {
+          ...col,
+          Cell: props => (
+            <Link
+              to={`/opo/${opos.find(opo => opo.name === props.value)?.opo}`}
+            >
+              {props.value}
+            </Link>
+          ),
+        };
+      } else if (
```

Comment:
don't need else if here because of the early return on line 34, right? Would prefer consecutive `if`s then (and below on 71)